test/helpers/transactions/CreateOrderTransaction.ts

```diff
@@ -0,0 +1,29 @@
+export const CreateOrderTransaction = {
```

Comment:
"CreateOrder" is not a transaction actually. And I guess this folder should be named "message" (this term includes transactions, messages, auth, customData, etc.), not "transactions"

Reply:
done